systemtest/src/test/java/io/strimzi/systemtest/log/LoggingChangeST.java

```diff
@@ -712,7 +713,10 @@ void testDynamicallySetConnectLoggingLevels() {
         final Pattern log4jPatternDebugLevel = Pattern.compile("^(?<date>[\\d-]+) (?<time>[\\d:,]+) DEBUG (?<message>.+)");
         final Pattern log4jPatternInfoLevel = Pattern.compile("^(?<date>[\\d-]+) (?<time>[\\d:,]+) INFO (?<message>.+)");
 
-        final KafkaConnect connect = KafkaConnectTemplates.kafkaConnect(testStorage.getClusterName(), testStorage.getNamespaceName(), 3)
+        // logging changes on multiple Connect instances are possible from Kafka 3.7.0
```

Comment:
Should we add a comment to remember that after we support Kafka 3.7.0 and greater then remove this conditional?

Reply:
Yeah good point, we have the similar check in `MigrationST`, so I will add comment to those places